lib/double_entry/reporting/aggregate.rb

```diff
@@ -25,7 +25,7 @@ def amount(force_recalculation = false)
         end
       end
 
-      def formatted_amount(value = amount())
+      def formatted_amount(value = amount)
```

Comment:
I think @joneslee85 hand changed this to `amount()` because of a circular argument reference warning or something